coriolis/tasks/replica_tasks.py

```diff
@@ -269,8 +264,12 @@ def _run(self, ctxt, instance, origin, destination, task_info,
         connection_info = base.get_connection_info(ctxt, origin)
 
         source_environment = task_info['source_environment'] or {}
+        volumes_info = self._get_volumes_info(task_info)
```

Comment:
Do please remember to add `volumes_info` in the `required_task_info_properties` above